evalml/automl/automl_search.py

```diff
@@ -494,9 +494,20 @@ def __init__(
             index_columns = list(
                 self.X_train.ww.select("index", return_schema=True).columns
             )
-            index_columns = _put_into_original_order(self.X_train, index_columns)
-            if len(index_columns) > 0 and drop_columns is None:
-                parameters["Drop Columns Transformer"] = {"columns": index_columns}
+            unknown_columns = list(
+                self.X_train.ww.select("Unknown", return_schema=True).columns
+            )
```

Comment:
Is it possible to select the unknown and index columns in one call?  I ask because after this next WW release, select will return to us the columns in their original order so that we can probably refactor out `_put_into_original_order`.  Merging the select of index and unknown gets us ready for that.